Pod/Classes/UITextView+URBNHighlightExtra.h

```diff
@@ -0,0 +1,15 @@
+//
+//  UITextView+URBNHighlightExtra.h
+//  Pods
+//
+//  Created by Nick DiStefano on 6/19/15.
+//
+//
+
+#import <UIKit/UIKit.h>
+
+@interface UITextView (URBNHighlightExtra)
+
++ (instancetype)urbn_UITextViewWithErrorColor:(UIColor *)errorColor maxLength:(NSInteger)maxLength;
```

Comment:
So I'm not a huge fan of this naming. I would prefer something a little more generic. Like `urbn_textViewWithHighlightColor:(UIColor *)highlightColor startingAtIndex:(NSInteger)startingIndex;`. Or something like that. The idea is that maybe its not an error. Also might be nice to augment this a little bit. Like the ability to create a generic version of these textfields and update their colors and index later. So you could have a few different colors. See below:

```
+ (instanceType)urbn_highlightTextField;
- (void)urbn_addHighlightColor:(UIColor *)highlightColor startingAtIndex:(NSInteger)startingIndex;

etc....
```